ez_setup.py

```diff
@@ -93,12 +94,12 @@ def do_download():
         pkg_resources.require("setuptools>="+version); return
     except pkg_resources.VersionConflict as e:
         if was_imported:
-            print >>sys.stderr, (
+            print((
```

Comment:
Nit: There are unnecessary parentheses around the format string now; the string is implicitly continued via being in the opening parens for `print(`.